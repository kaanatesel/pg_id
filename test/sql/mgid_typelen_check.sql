select typlen from pg_type where oid = 'mgid'::regtype::oid;

select pg_column_size(mgid '00000000bcf86cd799439011');