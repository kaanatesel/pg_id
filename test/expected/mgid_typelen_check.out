select typlen from pg_type where oid = 'mgid'::regtype::oid;
 typlen 
--------
     12
(1 row)

select pg_column_size(mgid '00000000bcf86cd799439011');
 pg_column_size 
----------------
             12
(1 row)

